index.html

```diff
@@ -655,7 +657,41 @@ <h4>LET'S CONNECT</h4>
 
          </div>
 
-     </section>
+     </div>
+
+     <div class="contacto">
+
+         <div class="flex-column fondo">
+
+             <h2 class="margen">Contact me</h2>
+
+             <p class="margen">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
+
+             <form action="https://formspree.io/f/mayzvpoz" method="post" class="margen">
+
+                 <input type="text" name="name" maxlength="30" required placeholder="Lorem Ipsum">
+
+                 <input type="email" required placeholder="loremipsum@gmail.com">
+
+                 <textarea maxlength="500" rows="4" cols="50" required placeholder="Write your message here..."></textarea>
+
+                 <div>
+                    
+                     <button id="boton" type="submit" >
+                        
+                         Get in touch
+
+                     </button>
+
+                 </div>
+
+             </form>
+
+             <div id="ocult"><hr class="ocultarmob"></div>
+
+         </div>
+
```

Comment:
- [ ] KIndly remove the white portion below your contact form :+1: 

| Your Design | The Figma Design |
|--------|--------|
| ![image](https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/assets/102699958/15b2e631-e811-4823-a090-3c154a61136a) | ![image](https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/assets/102699958/1d041bab-4cc9-47ca-91b2-dd76a8c641b0) |

Reply:
Done, thank you NahnahAJ 👍😃